### DISCUSSION TITLE
Buffs self only no matter what

### DISCUSSION THREAD

**Discussion by Super-huge** [smasherprog/EqTool#32] [Q&A]
I've been using EQTool for a number of weeks and I like it. Over the past few days / weeks (not sure since I was mostly soloing), the tool no longer tracks buffs that I cast on other players (PCs only). I've tried with the option "Effect Me Only" (it's unchecked, so I checked it then unchecked it), and also tried a reinstall. No matter what it's not picking up the spells I cast on PCs. Anyone else have this or find a resolution?

**Comment by smasherprog** [comment]
This is very strange.

So, buffs show on you, but no buffs show on anyone else?
hmmm

**Comment by smasherprog** [comment]
Do you have a REGULAR install of p99? 
No crazy mods?

**Comment by smasherprog** [comment]
This is now fixed. The problem was the class list in settings.
I changed this to a checkbox list so you can see which classes you have filters for

**Comment by smasherprog** [comment]
Open an issue it this doesnt fix it

**Comment by Super-huge** [comment]
Ok thanks. Sorry didn't get a chance to reply but no mods, standard UI even. It was working originally, and then it wasn't, without any notable changes on my part.

**Comment by smasherprog** [comment]
Check it again. It should be fixed. 
Open your settings and make sure the correct classes are selected
